### DISCUSSION TITLE
Support for Stopping at Target Build Stages

### DISCUSSION THREAD

**Discussion by ClaytonNorthey92** [testcontainers/testcontainers-node#619] [Ideas]
Hello! As far as I can tell, when building an image, `testcontainers-node` doesn't support [stopping at a build stage](https://docs.docker.com/build/building/multi-stage/#stop-at-a-specific-build-stage).  Am I correct? If so, I could use this feature on some of my projects.  I would be happy to add it.  I think we would need to expose [the `target` option](https://docs.docker.com/engine/api/v1.37/#tag/Image/operation/ImageBuild) in the `dockerode.buildImage` function where we can build images in `testcontainers-node`

**Comment by cristianrgreco** [comment]
Hi @ClaytonNorthey92, indeed testcontainers-node does not support that feature. Yes please a contribution would be welcome

**Comment by ClaytonNorthey92** [comment]
@cristianrgreco I created an issue #629  and a [PR](https://github.com/testcontainers/testcontainers-node/pull/628) for that issue 👍
